rasa/shared/core/training_data/story_reader/markdown_story_reader.py

```diff
@@ -53,7 +53,9 @@ def __init__(
                 docs=DOCS_URL_MIGRATION_GUIDE_MD_DEPRECATION,
             )
 
-    def read_from_file(self, filename: Union[Text, Path]) -> List[StoryStep]:
+    def read_from_file(
+        self, filename: Union[Text, Path], skip_validation: bool = False
```

Comment:
I added the `skip_validation` parameter here because I added it to the base class. I'm not using it here in `markdown_story_reader` so it might be a bit misleading. Happy to hear ideas on how to solve this!! 🤯

Reply:
cc @chdorner in case that's not aligned with the changes that you've planned 😬

Reply:
Thanks for the heads-up! We haven't been on such a detailed level yet, so all good! Looks like a worthwhile flag to add anyways for when we're sure that it's valid.